apps/wallet/src/screens/Authentication/Login/internal.ts

```diff
@@ -35,7 +36,11 @@ export const signIn = async (invitationCode?: string) => {
 		} else if (status === ThresholdResult.Missing) {
 			navigate('Authentication', { screen: 'Recovery' });
 		} else if (status === ThresholdResult.Ready) {
-			navigate('Dashboard');
+			if (appState.profile.email) {
+				navigate('Dashboard');
+			} else {
+				navigate('Authentication', { screen: 'Recovery' });
+			}
```

Comment:
I don't think this case could happen, can you explain this one

Reply:
This case happens when a user logs in, then logs out, and logs in again, the `status` is `ready` (the `requiredShares <= 0` and the `totalShares = 3`, it seems like our `storage.clearAllDocs()` func not works properly I guess, we can dive into it in another task). So in that case, we'll navigate directly to the `Dashboard` without storing any data.

Reply:
storage.clearAllDocs() only clears our storage usage, not including w3a. I think you need to resolve this one, it's an unexpected behavior and critical to resolve. Otherwise, I remember each user account has w3a metadata, this case might only happen when the signed in user is the one has just signed out. So you need to find a way that can clean w3a metadata of the signed user.

This flow needs to be clear, does not resolve by stack up the logic